tutorials/lodash/truncate.md

```diff
@@ -0,0 +1,38 @@
+The `truncate` function allows you to shorten a string if it is longer than a specifed length.
+It will shorten the string and replace the last characters of the shortened string with the characters you specify, or the default `...`.
+It takes three options:
+
+- length (number), the maximum length of the string.
+- omission (string), the string to indicate text is omitted.
+- separator (RegExp | string), the separator pattern to truncate to. 
```

Comment:
Let's create a separate section for the 'separator' argument, with a separate example. Putting all 3 params in one section is too much.